diffpy/pdffit2/pdffit.py

```diff
@@ -148,7 +148,7 @@ def intro():
         date = __date__[:10]
         d = {"version": __version__, "date": date, "year": date[:4] or "2019"}
         msg = __intro_message__ % d
-        filler = lambda mx: (mx.group(0).rstrip(" *").ljust(77) + "*")
+        filler = lambda mx: (mx.group(0).rstrip(" *").ljust(77) + "*")  # noqa: E731
```

Comment:
I think this lambda needs to be copy-pasted into the row below (and remove the noqa.  To keep it readable you can put a line-break in, so functionally it reads as easily as it does now.

Reply:
btw, I think what you did is fine too, though prefer my solution I think.  But Let's work on getting tests to pass with higher priority....